continuous_integration/environment-3.10.yaml

```diff
@@ -57,6 +57,9 @@ dependencies:
   - distributed
   - ipython
   - ipycytoscape
+  # until https://github.com/jupyter-widgets/ipywidgets/issues/3731 is fixed
+  - ipywidgets<7.7.4
```

Comment:
Any reason we need to drop the max version of `ipywidgets`? Looking through past runs, it seems like things were working up to 8.0.4:

https://github.com/dask/dask/actions/runs/4452937304/jobs/7821042159

Though I do think we'll need to set some max version, as using 8.0.5 I got failures:

```python-traceback
dask/base.py:800: in visualize
    return cytoscape_graph(dsk, filename=filename, **kwargs)
dask/dot.py:483: in cytoscape_graph
    g = ipycytoscape.CytoscapeWidget(
/datasets/charlesb/mambaforge/envs/upstream-observed-false/lib/python3.10/site-packages/ipycytoscape/cytoscape.py:859: in __init__
    super().__init__(**kwargs)
/datasets/charlesb/mambaforge/envs/upstream-observed-false/lib/python3.10/site-packages/ipywidgets/widgets/widget.py:501: in __init__
    super().__init__(**kwargs)
/datasets/charlesb/mambaforge/envs/upstream-observed-false/lib/python3.10/site-packages/traitlets/traitlets.py:1347: in __init__
    setattr(self, key, value)
/datasets/charlesb/mambaforge/envs/upstream-observed-false/lib/python3.10/site-packages/traitlets/traitlets.py:732: in __set__
    self.set(obj, value)
/datasets/charlesb/mambaforge/envs/upstream-observed-false/lib/python3.10/site-packages/traitlets/traitlets.py:706: in set
    new_value = self._validate(obj, value)
/datasets/charlesb/mambaforge/envs/upstream-observed-false/lib/python3.10/site-packages/traitlets/traitlets.py:738: in _validate
    value = self.validate(obj, value)
/datasets/charlesb/mambaforge/envs/upstream-observed-false/lib/python3.10/site-packages/ipywidgets/widgets/trait_types.py:360: in validate
    return super().validate(obj, self.klass(**value))
/datasets/charlesb/mambaforge/envs/upstream-observed-false/lib/python3.10/site-packages/ipywidgets/widgets/widget_layout.py:86: in __init__
    super().__init__(**kwargs)
/datasets/charlesb/mambaforge/envs/upstream-observed-false/lib/python3.10/site-packages/ipywidgets/widgets/widget.py:504: in __init__
    self.open()
/datasets/charlesb/mambaforge/envs/upstream-observed-false/lib/python3.10/site-packages/ipywidgets/widgets/widget.py:535: in open
    self.comm = create_comm(**args)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

args = ()
kwargs = {'buffers': [], 'data': {'buffer_paths': [], 'state': {'_model_module': '@jupyter-widgets/base', '_model_module_versio...el_name': 'LayoutModel', '_view_count': None, ...}}, 'metadata': {'version': '2.1.0'}, 'target_name': 'jupyter.widget'}

    def _create_comm(*args, **kwargs):
        """Create a Comm.
    
        This method is intended to be replaced, so that it returns your Comm instance.
        """
>       raise NotImplementedError("Cannot ")
E       NotImplementedError: Cannot

/datasets/charlesb/mambaforge/envs/upstream-observed-false/lib/python3.10/site-packages/comm/__init__.py:27: NotImplementedError
```

Reply:
@charlesbluca When testing things locally, I got errors with 8.0.5, and then dropping further, with 7.7.4. I'll retest again to make sure.

Reply:
Seems to work with 8.0.4. 🤞